addons/mod_tool/interface/file_system/file_system_context_actions.gd

```diff
@@ -183,6 +212,62 @@ static func add_script_extension_to_mod_main(file: File, extension_path: String)
 	ModToolUtils.output_info('Added script extension "%s" to mod "%s"' % [extension_path, main_script_path.get_base_dir().get_file()])
 
 
+func create_overwrite_asset(file_path: String) -> String:
+	if not ModToolStore.name_mod_dir:
+		ModToolUtils.output_error("Select an existing mod or create a new one to overwrite assets")
+		return ""
+
+	var file_directory := file_path.get_base_dir().trim_prefix("res://")
+	var overwrite_directory: String = ModToolStore.path_mod_dir.plus_file("overwrites").plus_file(file_directory)
+	ModToolUtils.make_dir_recursive(overwrite_directory)
+
+	var dir := Directory.new()
+	var overwrite_path := overwrite_directory.plus_file(file_path.get_file())
+	if not dir.file_exists(overwrite_path):
+		dir.copy(file_path, overwrite_path)
+		ModToolUtils.output_info('Copied asset "%s" as overwrite to path %s' % [file_path.get_file(), overwrite_path])
+
+	ModToolStore.editor_file_system.scan()
+	ModToolStore.editor_plugin.get_editor_interface().get_file_system_dock().navigate_to_path(overwrite_path)
+
+	return overwrite_path
+
+
+static func add_asset_overwrite_to_overwrites(vanilla_asset_path: String, asset_path: String) -> void:
+	var file := File.new()
+	var overwrites_script_path := ModToolStore.path_mod_dir.plus_file("overwrites.gd")
+	if not file.file_exists(overwrites_script_path):
+		file.open(overwrites_script_path, File.WRITE)
+		file.store_line("extends Node\n\n")
+		file.store_line("func _init() -> void:")
+		file.close()
+
+	if not script_has_method(overwrites_script_path, "_init"):
+		file.open(overwrites_script_path, File.READ_WRITE)
+		file.store_string(file.get_as_text() + "\nfunc _init() -> void:\n")
+		file.close()
+
+	file.open(overwrites_script_path, File.READ)
+	var file_content := file.get_as_text()
+	file.close()
+
+	# Construct the line required to preload the asset and take over the path
+	var asset_overwrite_line := "\tpreload(%s).take_over_path(%s)\n" % [quote_string(asset_path), quote_string(vanilla_asset_path)]
+
+	# Check if that asset is already being overwritten
+	if asset_overwrite_line.strip_edges() in file_content:
+		return
+
+	# If there is no modLoader.install_script_extension yet, put it at the end of install_script_extensions
```

Comment:
I don't get that comment why  `modLoader.install_script_extension` ?

Reply:
whoops, a leftover from duplicating